client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -245,10 +259,12 @@ const fixedCartModel = {
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
 			couponCode: couponCodeField,
-			fixedDiscount: {
-				...fixedDiscountField,
-				labelText: translate( 'Cart Discount', { context: 'noun' } ),
-			},
+			fixedDiscount: (
+				<CurrencyField
+					labelText={ translate( 'Cart Discount', { context: 'noun' } ) }
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.